gluetool/tool.py

```diff
@@ -256,10 +257,11 @@ def _handle_failure(self, failure):
             except Exception:
                 # tripple error \o/
 
-                print >> sys.stderr
-                print >> sys.stderr, '!!! While logging an exception, another exception appeared !!!'
-                print >> sys.stderr, '    Giving up on everything...'
-                print >> sys.stderr
+                print(file=sys.stderr)
+                # pylint: disable=line-too-long
+                print('!!! While submitting an exception to the Sentry, another exception appeared !!!', file=sys.stderr)
+                print(sys.stderr, '    Giving up on everything...', file=sys.stderr)
```

Comment:
Why do you use `sys.stderr` as a first parameter?

Reply:
Forgot to get rid of it. Fixed.